packages/convertkit/src/hooks/use-convertkit.tsx

```diff
@@ -29,8 +29,13 @@ const defaultConvertKitContext: ConvertkitContextType = {
 export const ConvertkitContext = React.createContext(defaultConvertKitContext)
 
 export const ConvertkitProvider: React.FC<
-  React.PropsWithChildren<{children: any}>
-> = ({children}) => {
+  React.PropsWithChildren<{getSubscriberApiUrl?: string}>
+> = ({
+  children,
+  getSubscriberApiUrl = process.env.NEXT_PUBLIC_CONVERTKIT_GET_SUBSCRIBER_URL ||
+    `/api/subscriber`,
```

Comment:
made these configurable which is what allows for the opt-in

Reply:
I think we should switch everything to use the skill api endpoints as a next step